src/OrchardCore.Modules/OrchardCore.Deployment/Controllers/ExportFileController.cs

```diff
@@ -52,7 +56,23 @@ public async Task<IActionResult> Execute(int id)
             {
                 archiveFileName = PathExtensions.Combine(Path.GetTempPath(), filename);
 
-                var deploymentPlanResult = new DeploymentPlanResult(fileBuilder);
+                var recipeDescriptor = new RecipeDescriptor();
+                CustomFileDeploymentStep customFileDeploymentStep = (CustomFileDeploymentStep)deploymentPlan.DeploymentSteps.FirstOrDefault(ds => ds.Name == "CustomFileDeploymentStep");
+
```

Comment:
`(CustomFileDeploymentStep)` will throw an exception if it doesn't exist I think
`as CustomFileDeploymentStep` should not throw and can then be null checked afterwards